geometry_msgs/Polygon ros_poly
int32 direccion_rep
bool reparado
